packages/labs/task/src/test/task_test.ts

```diff
@@ -67,11 +67,11 @@ suite('Task', () => {
       override update(changedProperties: PropertyValues): void {
         super.update(changedProperties);
         this.taskValue = this.task.value ?? this.task.error;
-        this.task.render({
+        this.task.render<string, string>({
```

Comment:
Ideally this could all be inferred from both the task definition and the callbacks. What's preventing that now?

Reply:
just talked with steve for a bit! and if i understand correctly, i agree :)

uploaded a change that infers types from the `StatusRenderer` provided by the user